scripts/lsync.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+if [ "$#" -ne 1 ] || ! [ -f "$1" ]; then
+  echo -e "Usage:\n\t$0 <PATH>" >&2
+  echo -e "Example:\n\t$0 content/zh/docs/concepts/_index.md" >&2
+  exit 1
+fi
+
+LOCALIZED="$1"
+
+# Try get the English version
+ENGSOURCE=`echo $LOCALIZED | sed "s/content\/..\//content\/en\//g"`
```

Comment:
Hi @tengqm . This script looks useful.
Could you add a brief description of what the script is checking.
lsync = check localization file synchronization?

What do you think about renaming the variable "ENGSOURCE".
When I read "ENGSOURCE", I first thought of engineering not English.
Could use the two char language code instead?
`EN_SRC`
`EN_SRC_PATH`

If the `en` file does not exist for a given localized file, then this indicates that the `en` version has moved or been removed?
However, if the `en` file path exists for a given localized file, confirm that the commit history is the same for both files (no updates to the `en` file). Is this correct?

Reply:
lsync - localization synchronization

var name can be changed, for clarity

If the `en` file is gone, the localized version should be removed as well.
If the `en` file does exists, we check if it has been modified since the last time we committed the localized version for that file. The `git diff ...` command shows you how the English source has evolved.

There is an assumption here, however. We assume that the localized file, when merged, is still in sync with the English version. This is a simplification, but still reasonable.